rpm.spec

```diff
@@ -514,12 +505,7 @@ autoreconf -i -f
 	--sharedstatedir=%{_var}/lib \
 	--with-archive \
 	--with-vendor=%{_real_vendor} \
-%ifarch %{oldarches}
-	--with-external-db \
 	--enable-bdb-ro \
-%else
-	--disable-bdb-ro \
-%endif
```

Comment:
Shouldn't `--enable-bdb-ro` be removed?

Reply:
No. It makes it possible to read bdb databases in chroots from other environments, which is quite useful for forensic purposes and such.

Reply:
The point of my commit was getting rid of bdb-ro on arches that never had a bdb backed rpm -- didn't relaize that bdb-ro somehow manages to access bdb databases without using libdb (probably through an internalized bdb version? Got to check on that...)

Reply:
The BDB datafile format was reimplemented from scratch for bdb-ro.

Reply:
Ah, okay.